common/src/main/java/com/wynntils/core/functions/FunctionManager.java

```diff
@@ -317,6 +317,10 @@ public static void init() {
         registerFunction(new CharacterFunctions.XpPctFunction());
         registerFunction(new CharacterFunctions.BpsFunction());
         registerFunction(new CharacterFunctions.BpsXzFunction());
+        registerFunction(new CharacterFunctions.MoneyFunction());
```

Comment:
Can you preserve alphabetical order? (forgot to do this for Bps too)

Reply:
I will get on that